EnglishFilter/sections/specific.txt

```diff
@@ -1218,7 +1219,7 @@ ndtv.com##.exam
 onlinecoursereport.com##.degree-finder-sidebar
 adgully.com##body > div[class^="asd"]
 gossipfunda.com##.widget_mc4wp_form_widget ~ aside.widget
-newforex.online##.demand-supply
+forexmab.com,newforex.online##.demand-supply
```

Comment:
Can you add `forex-gold.net` as well?

<details>
<summary>Screenshot</summary>

![forex-gold](https://user-images.githubusercontent.com/58900598/156607342-1e67d7de-ac69-45c2-8008-eea48b37788d.png)

</details>

It's uBO but AG should be the same in this case.